bindings/guile/core-utils.scm

```diff
@@ -38,8 +38,8 @@
 
 (define-public gnc:version (gnc-version))
 ;; gettext functions
-(define-public _ gnc:gettext)
+(define-public G_ gnc:gettext)
 (define-syntax N_
     (syntax-rules ()
-        ((_ x) x)))
+        ((G_ x) x)))
```

Comment:
No; in this `((_ x) x)))`, the `_` is part of macro syntax. It must be left alone. A more concise definition of `N_` is:
```scheme
(define-syntax-rule (N_ x) x)
```